app/controllers/name_controller.rb

```diff
@@ -1175,6 +1176,21 @@ def submit_tracking_form(name_id)
     redirect_with_query(action: "show_name", id: name_id)
   end
 
+  def notify_admins_of_notification(notification)
+    return if notification.note_template.blank?
+    return if !notification.new_record? &&
+              notification.note_template_before_last_save.blank?
```

Comment:
Does the method `note_template_before_last_save` already exist?

Reply:
Yes, it's an automagic method.  Before you save an ActiveRecord instance, you can access the original value of any attribute "attr" by tacking on "_before_last_save", that is "instance.attr_before_last_save".  (It used to be "attr_was", but that is now deprecated.)